experiment/build/generate_cloudbuild.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Generates Cloud Build specification"""
+
+import json
+import yaml
+
+from experiment.build import docker_images
+
+
+def create_cloud_build_spec(buildable_images):
+    """Returns Cloud Build specificatiion."""
+
+    cloud_build_spec = {}
+    cloud_build_spec['steps'] = []
+    cloud_build_spec['images'] = []
+
+    for name, image in buildable_images.items():
+        step = {}
+        step['id'] = name
+        step['name'] = 'gcr.io/cloud-builders/docker'
+        step['args'] = []
+        step['args'] += ['--tag', image['tag']]
+        step['args'] += ['--cache-from', 'gcr.io/fuzzbench/' + image['tag']]
```

Comment:
project name shouldn't be hardcoded, can we get the right one somehow.

Reply:
done